PP_APP/server/frontend/templates/post_view.html

```diff
@@ -0,0 +1,59 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+
+<section id="about" class="about-section bg-light">
+    <div class="container">
+        {% for key in pp_post %}
+            <h1>POST #{{key.img_post.id}}</h1>
+            <table class="table">
+                <tr>
+                    <th>User</th>
+                    <td>{{key.img_post.user_id.email}}</td>
+                </tr>
+                <tr>
+                    <th>Top</th>
+                    <td><a href="../../media/{{key.img_post.top_img}}" target="blank">{{key.img_post.top_img}}</a></td>
+                </tr>
+                <tr>
+                    <th>Side</th>
+                    <td><a href="../../media/{{key.img_post.side_img}}" target="blank">{{key.img_post.side_img}}</a></td>
+                </tr>
+                <tr>
+                    <th>Infer</th>
+                    <td><a href="../../media/{{key.img_post.infer_img}}" target="blank">{{key.img_post.infer_img}}</a></td>
+                </tr>
+                <tr>
+                    <th>Date</th>
+                    <td>{{key.img_post.date_posted}}</td>
+                </tr>
+                <tr>
+                    <th>Brand Name</th>
+                    <td>{{key.packager.brand_name}}</td>
+                </tr>
+                <tr>
+                    <th>Materials</th>
+                    <td>{{key.materials}}</td>
+                </tr>
+                <tr>
+                    <th>Score</th>
```

Comment:
Another thing for after the beta; tell them how we calculated the score. We can put a thorough explanation of the score in a page and put a link here so they can go and read about it. But this is great for now.